<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Budget Tracker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/icons.qrc">
    <normaloff>:/newPrefix/images/money.png</normaloff>:/newPrefix/images/money.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>645</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionLedgers"/>
    <addaction name="actionCategories"/>
    <addaction name="actionBudgets"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionImporting_CSVs"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuAnalyze">
    <property name="title">
     <string>Analyze</string>
    </property>
    <addaction name="actionCharts"/>
    <addaction name="actionSummary"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuAnalyze"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionLedgers">
   <property name="text">
    <string>Ledgers</string>
   </property>
   <property name="toolTip">
    <string>View ledger tables</string>
   </property>
  </action>
  <action name="actionCategories">
   <property name="text">
    <string>Categories</string>
   </property>
  </action>
  <action name="actionBudgets">
   <property name="text">
    <string>Budgets</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNew_Ledger">
   <property name="text">
    <string>New Ledger</string>
   </property>
  </action>
  <action name="actionImporting_CSVs">
   <property name="text">
    <string>Importing CSVs</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCharts">
   <property name="text">
    <string>Charts</string>
   </property>
  </action>
  <action name="actionSummary">
   <property name="text">
    <string>Summary</string>
   </property>
  </action>
  <action name="actionAdd_Ledger">
   <property name="text">
    <string>Add Ledger</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionDelete_Ledger">
   <property name="text">
    <string>Delete Ledger</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
